sdk/eventhub/event-hubs/test/utils/testUtils.ts

```diff
@@ -17,6 +17,7 @@ export const isNode =
 export enum EnvVarKeys {
   EVENTHUB_CONNECTION_STRING = "EVENTHUB_CONNECTION_STRING",
   EVENTHUB_NAME = "EVENTHUB_NAME",
+  EVENTHUB_FQDN = "EVENTHUB_FQDN",
```

Comment:
I recall removing this at one point as it can be inferred from the connection string. Any reason to add it back rather than doing the inferring?

Reply:
Ah, didn't know that. It's still in the samples.env but I can extract it (there's a function that kind of does it correctly that can be made to do what I need).

Reply:
Removed, parsed out of connection string instead.